Makefile

```diff
@@ -78,6 +78,9 @@ kind: make-cache out-dir
 		-e CGO_ENABLED=0 \
 		-e GOOS=$(GOOS) \
 		-e GOARCH=$(GOARCH) \
+		-e HTTP_PROXY=$(HTTP_PROXY) \
```

Comment:
this is a start, we can consolidate lowercase to these above later if absolutely necessary, but imho we should try to keep this very simple, I'm not aiming to grow a "build system", just a simple alternative to running go natively to avoid complaints about packaged go versions and give us a more reproducible option for releases :sweat_smile: